gaia/tasks.py

```diff
@@ -0,0 +1,42 @@
+from celery import Celery
+from gaia.parser import parse_request
+from gaia.core import config
+
+app = Celery('tasks',
+             backend=config['gaia']['celery_backend'],
+             broker=config['gaia']['celery_broker'])
+
+
+@app.task
+def parse_process(process_name, request_json):
+    """
```

Comment:
@mbertrand 
When you go to call ```parse_process.delay('within', {....})``` to actually call this function and have it execute in a worker,  it is going to create an underlying json object that gets published on the RabbitMQ exchange.  That message will include the name of the task (in this case ```gaia.tasks.parse_process```) and the arguments to this task (in this case 'within'  and the request_json). The arguments to the celery task are going to be a gaia task and its arguments. We will be essentially serializing every function twice - is there an up side to doing it this way that i'm just not seeing?